### DISCUSSION TITLE
[Bug]: Grenades don't stack

### DISCUSSION THREAD

**Discussion by Miner16** [Lorenzo0111/QualityArmory#570] [Q&A] [answered]
### Confirmations

- [X] I am using the latest plugin version with the required dependencies.
- [X] There isn't any open issue that shows my problem

### What happened?

Incendiary grenades, normal grenades, flashbangs, proxy mines, and smoke grenades don't stack.

### Expected Behaviour

They should stack as 'maxItemStack' is set to 10 in every config.

### Plugins

not relevant

### Server Version

1.16.5

**Comment by Lorenzo0111** [comment, marked as answer]
This is because crossbows can't be stacked. Change the material and it will be doable

**Comment by Miner16** [comment]
What item should i change it to?

**Comment by Lorenzo0111** [comment]
for example a stick

**Comment by Miner16** [comment]
Sorry for the question bombing but what do I have to change in the texturepack to make them appear on the stick?

**Comment by Lorenzo0111** [comment]
you need to move the part that mentions that gun from the crossbow.json to the stick.json

**Comment by Lorenzo0111** [comment]
@Miner16 if you'll still need help you can ask here
